include/fmt/printf.h

```diff
@@ -474,8 +474,18 @@ OutputIt basic_printf_context<OutputIt, Char>::format() {
   const Char* end = parse_ctx_.end();
   auto it = start;
   while (it != end) {
+    if (end - it < 32) {
+      // Use a simple linear search instead of memchr for small strings.
+      it = std::find(it, end, '%');
+    } else {
```

Comment:
I don't think this branch is necessary. `std::find` and `defail::find` will likely provide an equivalent code. I suggest removing it.

Reply:
Happy to do so.